Feature/dev 545

iolesyk opened the pull request:
* Add support Ubuntu 22.04
* Add support Red5 Pro Cloud storage plugin configuration to record streams to the AWS S3 bucket
* CORS update

ravisrivastva-qasource commented on variables.tf:
```diff
@@ -12,7 +12,7 @@ variable "type" {
   default     = ""
   validation {
-    condition     = var.type == "single" || var.type == "cluster" || var.type == "autoscaling"
+    condition     = var.type == "single" || var.type == "cluster" || var.type == "autoscaling" || var.type == "vpc"
```
@iolesyk There is no use of `var.type == "vpc"` as a validation condition, because we only have to validate the variable values with `single`, `cluster` and `autoscaling`.

ravisrivastva-qasource reviewed (commented):
@iolesyk Kindly modify the `r5p_install_server_basic.sh` script to add the support for installing JDK in ubuntu version `22.04`. 
https://github.com/red5pro/terraform-aws-red5pro/blob/d12c64392b34449c11f787607f03eafa13d6f14b/red5pro-installer/r5p_install_server_basic.sh#L56C33-L56C33

Also, I found in `r5p_create_node_group.sh` script there is the use of `curl` command inside the `curl` for verifying the `http_code` respose of the URL.
https://github.com/red5pro/terraform-aws-red5pro/blob/d12c64392b34449c11f787607f03eafa13d6f14b/red5pro-installer/r5p_create_node_group.sh#L97C53-L97C57

iolesyk commented:
* Add support ubuntu 22.04 in the script: r5p_install_server_basic.sh
* Updated logic to check Stream Manager in the script r5p_create_node_group.sh, deleted extra curl usage
* Updated examples, added red5pro cloud storage configuration blocks
* Updated README
* Monor fixes
